api-common-java/src/test/java/com/google/api/core/ApiFuturesTest.java

```diff
@@ -38,12 +38,12 @@
 import java.util.concurrent.ExecutionException;
 import java.util.concurrent.Executor;
 import java.util.concurrent.atomic.AtomicInteger;
-import org.junit.Test;
+import org.junit.jupiter.api.Test;
 
 public class ApiFuturesTest {
 
   @Test
-  public void testAddCallback() throws Exception {
+  public void testAddCallback() {
```

Comment:
You can remove `public` since this is not required in Junit 5.

Reply:
Updated.